src/pyhf/infer/test_statistics.py

```diff
@@ -30,7 +30,8 @@ def qmu(mu, data, pdf, init_pars, par_bounds):
         >>> test_mu = 1.0
         >>> init_pars = model.config.suggested_init()
         >>> par_bounds = model.config.suggested_bounds()
-        >>> pyhf.infer.test_statistics.qmu(test_mu, data, model, init_pars, par_bounds)
+        >>> q_mu = pyhf.infer.test_statistics.qmu(test_mu, data, model, init_pars, par_bounds)
+        >>> print(q_mu)
```

Comment:
Doctest was being weird and breaking with

```
=================================== FAILURES ===================================
___________________ [doctest] pyhf.infer.test_statistics.qmu ___________________
024         >>> pyhf.set_backend("numpy")
025         >>> model = pyhf.simplemodels.hepdata_like(
026         ...     signal_data=[12.0, 11.0], bkg_data=[50.0, 52.0], bkg_uncerts=[3.0, 7.0]
027         ... )
028         >>> observations = [51, 48]
029         >>> data = pyhf.tensorlib.astensor(observations + model.config.auxdata)
030         >>> test_mu = 1.0
031         >>> init_pars = model.config.suggested_init()
032         >>> par_bounds = model.config.suggested_bounds()
033         >>> pyhf.infer.test_statistics.qmu(test_mu, data, model, init_pars, par_bounds)
Expected:
    3.938244920380498
Got:
    array([3.93824492])
```

even though

```python
>>> import pyhf
>>> pyhf.set_backend("numpy")
>>> model = pyhf.simplemodels.hepdata_like(
...     signal_data=[12.0, 11.0], bkg_data=[50.0, 52.0], bkg_uncerts=[3.0, 7.0]
... )
>>> observations = [51, 48]
>>> data = pyhf.tensorlib.astensor(observations + model.config.auxdata)
>>> test_mu = 1.0
>>> init_pars = model.config.suggested_init()
>>> par_bounds = model.config.suggested_bounds()
>>> test_stat = pyhf.infer.test_statistics.qmu(test_mu, data, model, init_pars, par_bounds)
>>> print(f"test_stat: {test_stat} is of type {type(test_stat)}")
test_stat: 3.938244920380498 is of type <class 'numpy.float64'>
```

so I went ahead and used print to force it to accept the float.